ttnn/cpp/ttnn/operations/data_movement/slice/slice.cpp

```diff
@@ -2,23 +2,23 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-
 #include "ttnn/common/constants.hpp"
 #include "slice.hpp"
 #include "device/slice_op.hpp"
 #include "ttnn/run_operation.hpp"
 #include "ttnn/operations/core/core.hpp"
 #include "ttnn/cpp/ttnn/operations/creation.hpp"
 #include "ttnn/common/constants.hpp"
+#include "ttnn/cpp/ttnn/operations/data_movement/copy/copy.hpp"
 
 
 namespace ttnn::operations::data_movement {
 namespace detail {
-    uint32_t wrap_index(int index, int size) {
+    static inline uint32_t wrap_index(int index, int size) {
         return index < 0 ? size + index : index;
     }
-    uint32_t round_up_to_multiple_of_32(uint32_t value) {
-        return value == 0 ? 32 : ((value + 31) & ~31);
+    static inline uint32_t round_up_to_multiple_of_32(uint32_t value) {
+        return value == 0 ? 32u : ((value + 31u) & ~31);
```

Comment:
We should have this in utils somewhere

Reply:
adding it in a transpose change in another PR